src/Behat/Behat/Output/Statistics/TotalStatistics.php

```diff
@@ -175,9 +175,9 @@ public function registerHookStat(HookStat $stat)
     /**
      * Returns counters for different scenario result codes.
      *
-     * @return array[]
+     * @return int[]
```

Comment:
I suggest documenting this as `array<TestResults::*, int>` to be more precise about the array keys.